packages/amplify-cli/src/utils/readJsonFile.ts

```diff
@@ -1,19 +1,18 @@
 import fs from 'fs-extra';
-import parseJson from 'parse-json';
```

Comment:
There is a bug which we fixed by parseJSON - please don't directly use JSON.parse() as it leads to errors in windows. Please add this dependency back CC: @yuth

Reply:
will change back and keep parse-json